csv-data-management/info.json

```diff
@@ -12,7 +12,7 @@
     {
       "title": "Extract Data from Single CSV",
       "operation": "extract_data_from_csv",
-      "description": "This action will extract data from CSV based on column names provided. User can select option to deduplicate rows based on columns",
+      "description": "This action will extract data from CSV based on column names provided. User can select option to deduplicate values based on column(s) provided",
```

Comment:
How about this --> This action will extract data from CSV file based on column name user input. Optionally, user can also perform record deduplication based on user input column(s)

Reply:
Reworded and updated